### DISCUSSION TITLE
Bricks Builder

### DISCUSSION THREAD

**Discussion by ghost** [hizzle-co/noptin#819] [Ideas]
Please add integration with Bricks Builder's form.

Or a way I could add a custom action to create a subscriber when the form a form is submitted.

Thank you.

**Comment by picocodes** [comment]
We're working on this.

**Reply by ghost** [reply]
Amazing! By chance, do you have an ETA? I am currently deciding which marketing plugin to purchase.

**Reply by picocodes** [reply]
This is now live:- https://noptin.com/integrations/bricks-marketing-automation/
